shared/locales/de/website-donate.json

```diff
@@ -1,12 +1,16 @@
 {
 	"metadata": {
-		"title": "Mein Beitrag | Social Income"
+		"title": "Rebuilding Lives With Ismatu",
```

Comment:
Absolutely, that would be the ideal scenario. However, I'm not sure how easy this change would be since, as of now, metadata are pulled from the language files rather than from Firestore.

The current solution was merely a quick and temporary fix - as personalized metadata is definitely crucial for her social media campaign (trust towards her followers, who don't know SI, but her face). Essentially, after Ismatu's campaign concludes, I plan to revert to the generic metadata—I've even added it to my to-do list ;-).

I'm fine with either option.

Reply:
I pushed a commit, where you can add the campaign specific metadata in the admin part of the campaign. Does that work for you? 
<img width="690" alt="image" src="https://github.com/socialincome-san/public/assets/1204548/e9889e4d-b7f5-457e-91ea-d0f260ffad1e">